kate/src/com.rs

```diff
@@ -264,6 +267,48 @@ pub fn extend_data_matrix(
 	Ok(chunk_elements)
 }
 
+#[cfg(feature = "alloc")]
+pub fn par_extend_data_matrix(
```

Comment:
Do we want to keep old `extend_data_matrix`?

Reply:
I don't think we need, we can go with parallel one!